catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/api/FlatFilterBuilder.java

```diff
@@ -45,7 +45,9 @@
 
   FlatFilterBuilder beginBinaryComparisonType(String operator);
 
-  FlatFilterBuilder beginPropertyIsLikeType(String operator, boolean matchCase);
+  FlatFilterBuilder beginPropertyIsLikeType(String operator);
+
+  FlatFilterBuilder beginPropertyIsILikeType(String operator);
```

Comment:
What does the `I` stand for?

Reply:
"Insensitive" ... i.e. "Case Insensitive LIKE"